src/lib/operators/jit_operator/operators/jit_read_tuples.cpp

```diff
@@ -107,8 +107,6 @@ void JitReadTuples::before_specialization(const Table& in_table, std::vector<boo
 
   if (in_table.chunk_count() == 0) return;
 
-  // TODO(all) Should we create a get_first_valid_chunk() function
-  //  returning the first non-nullpointer-chunk of the table?
```

Comment:
Apparently, we don’t need that helper function anywhere else. We do, however, need the check. Currently, this would fail if the first chunk is non-existent.